staging/src/k8s.io/apimachinery/pkg/apis/meta/internalversion/register.go

```diff
@@ -63,7 +63,12 @@ func addToGroupVersion(scheme *runtime.Scheme, groupVersion schema.GroupVersion)
 	)
 	scheme.AddKnownTypes(SchemeGroupVersion,
 		&ListOptions{},
+		&metav1.GetOptions{},
```

Comment:
We convert labels and fields into their internal form once, rather than having to parse it.  The storage paths use the selectors.